README.md

```diff
@@ -203,12 +231,67 @@ One good way to automate is using CloudFormation, using this example snippet:
 }
 ```
 
-#### Elastic Beanstalk Installation ####
+#### For Elastic Beanstalk ####
 
 * In order to add tags to existing Elastic Beanstalk environment, you will
   need to rebuild the environment with the spot-enabled tag. Follow this
   [guide](http://www.boringgeek.com/add-or-update-tags-on-existing-elastic-beanstalk-environments)
 
+### Configuration of autospotting ###
+#### Default configuration ####
```

Comment:
I would phrase this in such a way to make it clear that this is for manual execution, mainly for testing and evaluation, as compared to the execution from the Lambda runtime, which is actually the default operation mode. That's where it makes sense to be able to specify a certain region, in order to limit the impact when testing it. 

At the moment, when executed for Lambda we can't pass any of the options available as flags when executed locally, so it is processing all available regions and we can't pass the options for on-demand capacity, since none of that is  configurable for Lambda yet.

That could be implemented using CloudFormation stack parameters, which could be used to generate a certain custom CloudWatch event containing global options, that would need to be parsed and integrated with our current configuration mechanism.